pkg/media/engine.go

```diff
@@ -0,0 +1,112 @@
+package media
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/pion/sdp/v2"
+	"github.com/pion/webrtc/v2"
+)
+
+const (
+	mediaNameAudio = "audio"
+	mediaNameVideo = "video"
+	// iOSH264Fmtp       = "level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f"
+	// firefoxH264Fmtp97 = "profile-level-id=42e01f;level-asymmetry-allowed=1"
```

Comment:
I'm not sure if we need these anymore. Since we are using `fmtp` specified in the offer sdp